### ISSUE TITLE
No command specified

### ISSUE THREAD

**Issue by JonnyMako** [swaywm/swayidle#62]
With the latest commit (7af7e37e2472dd9f48a34e5c7ff920ff9b29ae4f), when I issue:
```
swayidle timeout 30 'swaylock -c 000000' timeout 60 'swaymsg "output * dpms off"' resume 'swaymsg "output * dpms on"' before-sleep 'swaylock -c 000000'
```
I get:
```
No command specified! Nothing to do, will exit
```

**Comment by JonnyMako** [comment]
Just want to add that it works fine if I use config file, but surely one also wants to be able to do it inline?

**Comment by emersion** [comment]
Is this fixed by f6ebf4acd6d94867c8b3f6944ede25ae75fc8ac1?

**Comment by JonnyMako** [comment]
> Is this fixed by [f6ebf4a](https://github.com/swaywm/swayidle/commit/f6ebf4acd6d94867c8b3f6944ede25ae75fc8ac1)?

Yes, thanks!
